pkgs/by-name/gh/ghstack/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "ghstack";
+  version = "v0.9.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ezyang";
+    repo = "ghstack";
+    rev = "0c412bf";
```

Comment:
```suggestion
    rev = "0c412bf0d9515e11b58cddaeb1b1d9f0b17a5295"; # no tags
```

Reply:
Thank you for the suggestion. I've updated my commit.